subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan.c

```diff
@@ -287,7 +293,8 @@ void lll_scan_prepare_connect_req(struct lll_scan *lll, struct pdu_adv *pdu_tx,
 		*conn_space_us = conn_offset_us;
 		pdu_tx->connect_ind.win_offset = sys_cpu_to_le16(0);
 	} else {
-		uint32_t win_offset_us = lll->conn_win_offset_us;
+		uint32_t win_offset_us = lll->conn_win_offset_us +
+					 CONN_SPACING;
```

Comment:
Instead, the addition should be done here: https://github.com/zephyrproject-rtos/zephyr/blob/4801cbd921cd3148ff48c150ea3f8be79f657422/subsys/bluetooth/controller/ll_sw/ull_sched.c#L165

Reply:
Yes, good point - I didn't want to impact the connection update and scanning, as change was intended for central connection establishment only, mostly for simplicity and to serve our use case. I'm aware that connection updates would not preserve the spacing, which was intentional.
If you believe it makes sense, and adds value to do this throughout the advanced scheduling feature, I think that's what we should do.

Reply:
You are right, `ull_sched_after_mstr_slot_get` is used for placing the scan window after central connections, too.